latex.bzl

```diff
@@ -1,26 +1,39 @@
-def _latex_pdf_impl(ctx):
+LatexOutputInfo = provider(fields = ['format', 'file'])
+
+def _latex_impl(ctx):
     toolchain = ctx.toolchains["@bazel_latex//:latex_toolchain_type"].latexinfo
     custom_dependencies = []
     for srcs in ctx.attr.srcs:
         for file in srcs.files.to_list():
             if file.dirname not in custom_dependencies:
                 custom_dependencies.append(file.dirname)
     custom_dependencies = ','.join(custom_dependencies)
+
+    ext =".{}".format(ctx.attr.format)
```

Comment:
Nit: should this not include the `.`? (I'm mostly wondering about the `LatexOutputInfo` using this below on line 49.)

Reply:
I agree!